app/controllers/account/users_controller.rb

```diff
@@ -0,0 +1,46 @@
+class Account::UsersController < Account::AccountController
+  
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    organization = Organization.create(org_params)
+    user = organization.users.build(user_params)
```

Comment:
You don`t save the result. This code just create an object with data but does not save data to database. Add condition to check -> 
if @user.save... 
  do something 
else
  show page with form again
end